File: Print adjacency list[5-September-2023]/Solution.java
//  >> JAVA CODE <<

class Solution {
    public List<List<Integer>> printGraph(int V, int edges[][]) {
        List<List<Integer>> adj=new ArrayList<>();
        
        for(int i=0;i<V;i++)
        {
            adj.add(new ArrayList<>());
        }
        
        for(var edge:edges)
        {
            int u=edge[0],v=edge[1];
            adj.get(u).add(v);
            adj.get(v).add(u);
        }
        
            return adj;
    }
}

File: Print adjacency list[5-September-2023]/Solution.py
#   >> PYTHON CODE <<

class Solution:
    def printGraph(self, V : int, edges : List[List[int]]) -> List[List[int]]:
        adj = [[] for _ in range(V)]

        for edge in edges:
            u, v = edge[0], edge[1]
            adj[u].append(v)
            adj[v].append(u)

        return adj